drivers/spi/spi_gd32.c

```diff
@@ -42,6 +42,9 @@ struct spi_gd32_config {
 	uint32_t reg;
 	uint32_t rcu_periph_clock;
 	const struct pinctrl_dev_config *pcfg;
+#if IS_ENABLED(CONFIG_SPI_GD32_INTERRUPT)
```

Comment:
In this case, I think `#ifdef` is more clear. we just want to import the code if config is `defined`.

Reply:
Replace #if IS_ENABLED to #ifdef.